<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Czateria</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsrc.qrc">
    <normaloff>:/icons/czateria.png</normaloff>:/icons/czateria.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Login type</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="noNicknameRadioButton">
         <property name="text">
          <string>&amp;Generate nickname</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="nicknameOnlyRadioButton">
         <property name="text">
          <string>Nickna&amp;me only</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="nickAndPassRadioButton">
         <property name="text">
          <string>Nic&amp;kname and password</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Credentials</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="nicknameLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="maxLength">
          <number>16</number>
         </property>
         <property name="placeholderText">
          <string>Nickname</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="passwordLineEdit">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="readOnly">
          <bool>false</bool>
         </property>
         <property name="placeholderText">
          <string>Password</string>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="saveCredentialsCheckBox">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Save credentials (in plaintext)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QTableView" name="tableView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>491</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuNotification_style">
     <property name="title">
      <string>Notification style</string>
     </property>
     <addaction name="actionMessage_box"/>
     <addaction name="actionNative"/>
    </widget>
    <addaction name="actionSave_pictures_automatically"/>
    <addaction name="actionUse_emoji_icons"/>
    <addaction name="actionDiscard_unaccepted_messages"/>
    <addaction name="menuNotification_style"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionSave_pictures_automatically">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Save pictures automatically</string>
   </property>
   <property name="statusTip">
    <string>Save the received images into the pictures directory without displaying them</string>
   </property>
  </action>
  <action name="actionUse_emoji_icons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Use emoji icons</string>
   </property>
   <property name="statusTip">
    <string>Convert icons and emoticons to emoji</string>
   </property>
  </action>
  <action name="actionMessage_box">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Message box</string>
   </property>
   <property name="statusTip">
    <string>Use a message box to notify about new private conversations</string>
   </property>
  </action>
  <action name="actionNative">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Native</string>
   </property>
   <property name="statusTip">
    <string>Use desktop notifications for new private conversations</string>
   </property>
  </action>
  <action name="actionDiscard_unaccepted_messages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Discard unaccepted messages</string>
   </property>
   <property name="statusTip">
    <string>Discards messages from unaccepted conversations when partner disconnects</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="rsrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>noNicknameRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>nicknameLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>noNicknameRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>306</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nicknameOnlyRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>nicknameLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nicknameOnlyRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLineEdit</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nickAndPassRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>nicknameLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>342</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nickAndPassRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordLineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>102</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>469</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>nickAndPassRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>saveCredentialsCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>346</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
